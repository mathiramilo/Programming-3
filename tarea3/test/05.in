# un reinicio al principio
leer_datos
5 10 2 1 .
leer_rendimiento
10 3 2 1 .
max_datos_procesados
planificacion_optima
Fin