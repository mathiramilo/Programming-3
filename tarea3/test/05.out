1>2>3>4>Max Datos Procesados: 13
5>Cantidad Datos Planificacion: 13
6>Fin.
